packages/lit-element/src/lit-element.ts

```diff
@@ -273,6 +273,8 @@ export const _$LE = {
   },
   // eslint-disable-next-line
   _$changedProperties: (el: LitElement) => (el as any)._$changedProperties,
+  // eslint-disable-next-line
+  _$controllers: (el: LitElement) => (el as any)._$controllers,
```

Comment:
hm, for these three I think we want to give them explicit return type annotations, because otherwise the return type is `any`